FrejaEidClient/src/main/java/com/verisec/frejaeid/client/service/AuthenticationService.java

```diff
@@ -101,4 +119,30 @@ private boolean isPollingTimeExpired(long pollingEndTime) {
         return (System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime;
     }
 
+    public static byte[] readAllBytes(InputStream inputStream) throws IOException {
+        final int bufLen = 4 * 0x400; // 4KB
```

Comment:
Hex numbers do look interesting, but I woud argue that simple 1024 would be much more readable by human eye.
Also, you can name variable something like "bufferLengthInKB", which would make it even clearer what are the units.

Reply:
Done 😄